### PR TITLE
Remove required values from policy

### PR DESCRIPTION
**Pull request by msl8r**

### CODE REVIEW

File: policies/tagging/policy.json [hunk 1]
```diff
@@ -32,9 +26,7 @@
         },
         "allowedBusinessAreaNames": {
           "defaultValue": [
-            "CFT",
```
Comment (timja) [inline policies/tagging/policy.json:35]: these are very static and should still be validated

File: policies/tagging/policy.json [hunk 2]
```diff
@@ -87,16 +79,16 @@
...
                 {
                   "field": "tags['businessArea']",
-                  "notIn": "[parameters('allowedBusinessAreaNames')]"
+                  "in": "[parameters('allowedBusinessAreaNames')]"
                 },
                 {
                   "field": "tags['application']",
-                  "notIn": "[parameters('allowedApplicationNames')]"
-                }
+                  "in": "[parameters('allowedApplicationNames')]"
```
Comment (timja) [inline policies/tagging/policy.json:90]: why not use exists?

File: policies/tagging/policy.json [hunk 3]
```diff
@@ -16,13 +16,7 @@
...
         "allowedEnvironmentNames": {
           "defaultValue": [
-            "sandbox",
-            "development",
-            "testing",
-            "demo",
-            "ITHC",
-            "staging",
-            "production"
+            ""
```
Comment (timja) [inline policies/tagging/policy.json:19]: these are static and should still be validated

File: policies/tagging/policy.json [hunk 4]
```diff
@@ -44,14 +44,14 @@
         },
         "allowedApplicationNames": {
```
Comment (timja) [inline policies/tagging/policy.json:45]: this should be removed, all application names are allowed

File: policies/tagging/policy.json [hunk 5]
```diff
@@ -87,15 +87,15 @@
                 },
                 {
                   "field": "tags['environment']",
-                  "notIn": "[parameters('allowedEnvironmentNames')]"
+                  "in": "[parameters('allowedEnvironmentNames')]"
```
Comment (timja) [inline policies/tagging/policy.json:90]: why have these been swapped from notIn to in this doesn't seem right? ?

Comment (madjava) [reply policies/tagging/policy.json]: ...reverting back to original logic and condition

File: policies/tagging/policy.json [hunk 6]
```diff
@@ -92,10 +82,6 @@
                 {
                   "field": "tags['businessArea']",
                   "notIn": "[parameters('allowedBusinessAreaNames')]"
-                },
-                {
-                  "field": "tags['application']",
```
Comment (timja) [inline policies/tagging/policy.json:97]: we need to validate this tag exists but we don't care what's in it

File: policies/tagging/policy.json [hunk 7]
```diff
@@ -34,24 +34,14 @@
           "defaultValue": [
             "CFT",
             "crime",
-            "cross-cutting"
+            "Cross-Cutting"
```
Comment (msl8r) [inline policies/tagging/policy.json:37]: should this be all lowercase?

Comment (madjava) [reply policies/tagging/policy.json:37]: The value is  cap for first letter in the [common module](https://github.com/hmcts/terraform-module-common-tags/blob/c4c3d2064c30f3bd1aeffa9cf50da315d07132fa/team-config.yml#L14) so updated here as well, my assumption is its case sensitive

File: policies/tagging/policy.json [hunk 8]
```diff
@@ -81,7 +81,7 @@
                 },
                 {
                   "field": "tags['businessArea']",
-                  "in": "[parameters('allowedBusinessAreaNames')]"
+                  "notIn": "[parameters('allowedBusinessAreaNames')]"
```
Comment (madjava) [inline policies/tagging/policy.json:84]: 👌🏾  ...missed that

**Review by madjava — APPROVED** [review]

File: policies/tagging/README.md [hunk 9]
```diff
@@ -28,6 +25,3 @@ businessArea:
  - "CFT"
  - "crime"
  - "cross-cutting"
- 
- builtFrom:
```
Comment (timja) [inline policies/tagging/README.md:32]: why is this removed from the docs?

File: policies/tagging/README.md [hunk 10]
```diff
@@ -12,9 +12,6 @@ All environments and resources:
 ## Allowed values
 Tags and their allowed values.
 
-application:
```
Comment (timja) [inline policies/tagging/README.md:15]: this should still be included, required, an approved application name: see https://github.com/hmcts/terraform-module-common-tags/blob/master/team-config.yml#L13

File: policies/tagging/README.md [hunk 11]
```diff
@@ -12,6 +12,9 @@ All environments and resources:
 ## Allowed values
 Tags and their allowed values.
 
+application:
+ - "core"
```
Comment (timja) [inline policies/tagging/README.md]: core is not correct, this should link to the data dictionaries, https://github.com/hmcts/terraform-module-common-tags/blob/master/team-config.yml and https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml

**Review by timja — APPROVED** [review]
